app.py

```diff
@@ -1367,6 +1379,9 @@ def hour_of_code(level, program_id=None):
 @app.route('/hedy/<int:level>', methods=['GET'], defaults={'program_id': None})
 @app.route('/hedy/<int:level>/<program_id>', methods=['GET'])
 def index(level, program_id):
+    if utils.is_redesign_enabled():
+        return tryit(level, program_id)
```

Comment:
Great! very easy and painless to do for now until we decide to get rid of old code!